doc/howto_release.md

```diff
@@ -29,56 +71,55 @@ git show
 git push upstream
 ```
 
-### Create release tag
-
-(For background, see <https://help.github.com/en/articles/creating-releases>)
-
-Preparation:
+## Create variables
 
-#### Changelog and tagging etc preparations
+For convenience, create Bash variables with the version update script:
 
 ```bash
-# update from GH
-#  assumptions:
-#  - own fork as "origin"
-#  - remote repo as "upstream"
-git fetch --all --prune && git checkout releasebranch_8_2 && \
- git merge upstream/releasebranch_8_2 && git push origin releasebranch_8_2
-
-# create version env var for convenience:
 # Get VERSION and TAG as variables.
-eval `./update_version.py status --bash`
+eval $(./update_version.py status --bash)
 ```
 
-#### Tag release (on GitHub)
-
 Version and tag are the same for all releases:
 
 ```bash
 echo "$VERSION"
 echo "$TAG"
 ```
 
-To be done in GH interface:
+If in doubt, run it without `eval $(...)` to see all the variables created.
```

Comment:
```suggestion
If in doubt, run `update_version.py` without `eval $(...)` to see all the variables created.
```

Reply:
The path to it was actually missing. Now fixed in d425132ed3 and the text says _the script_.